db/config.cc

```diff
@@ -416,16 +416,16 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     */
     , commit_failure_policy(this, "commit_failure_policy", value_status::Unused, "stop",
         "Policy for commit disk failures:\n"
-        "* die          Shut down gossip and Thrift and kill the JVM, so the node can be replaced.\n"
-        "* stop         Shut down gossip and Thrift, leaving the node effectively dead, but can be inspected using JMX.\n"
+        "* die          Shut down gossip and kill the JVM, so the node can be replaced.\n"
```

Comment:
What JVM? :-)

Reply:
lol. Just another Very strong Monster. since we are here, will drop it.